ModelViewer/ModelViewerTest/test.cpp

```diff
@@ -231,6 +235,32 @@ void ModelViewerTest::loadModel()
 	QVERIFY(loaded.m_isValid);
 }
 
+void ModelViewerTest::loadShader()
+{
+	bool success = m_pWindow->GetGraphicsWindow()->loadVertexShader("../Data/Shaders/ads.vert");
+	QVERIFY(success);
+	success = m_pWindow->GetGraphicsWindow()->loadFragmentShader("../Data/Shaders/ads.frag");
+	QVERIFY(success);
+}
+
+void ModelViewerTest::loadCurrentShaders()
+{
+	bool success = m_pWindow->GetGraphicsWindow()->reloadCurrentShaders();
+	QVERIFY(success);
+}
+
+void ModelViewerTest::editCurrentShaders()
+{
+	bool success = m_pWindow->GetGraphicsWindow()->editCurrentShaders();
```

Comment:
Does this open notepad when tests are run?  If so, please make a way to disable this test.